### PR TITLE
refactor clip and tokenizer for: v2, v1, wukong, cnclip

### PR DESCRIPTION
**Pull request by DongyaoZhu**

### CODE REVIEW

File: examples/stable_diffusion_v2/train_text_to_image.py [hunk 1]
```diff
@@ -112,18 +112,39 @@ def load_pretrained_model_clip_and_vae(pretrained_ckpt, net):
...
-    dataset = build_dataset(args, rank_id, device_num)
-
     # build model
     latent_diffusion_with_loss = build_model_from_config(args.model_config)
     pretrained_ckpt = os.path.join(args.pretrained_model_path, args.pretrained_model_file)
-    load_pretrained_model(pretrained_ckpt, latent_diffusion_with_loss)
+    if "cnclip" in args.model_config:
+        load_pretrained_model_vae_unet_cnclip(pretrained_ckpt, latent_diffusion_with_loss)
+    else:
+        load_pretrained_model(pretrained_ckpt, latent_diffusion_with_loss)
```
Comment (SamitHuang) [inline examples/stable_diffusion_v2/train_text_to_image.py:143]: 1. `"cnclip" in args.model_config` , it is unreliable to use file name mining to tell the type of clip.
2. knowing that these CLIP models share the same arch but with different vocab_size/width/layers config, it should be feasible to unify one `load_model` function to create different CLIP models

**Comment by SamitHuang** [comment]
Please also update readme to show user how to use SD 1.5. Thanks.

File: examples/stable_diffusion_v2/README.md [hunk 2]
```diff
@@ -129,18 +129,42 @@ DreamBooth allows users to generate contextualized images of one subject using j
...
+### 6. Evaluation
 
 Please refer to [Evaluation for Diffusion Models](eval/README.md)
 
 - - -
 ## Stable Diffusion 1.x
 
+We provide [SD1.5 weights](https://download.mindspore.cn/toolkits/mindone/stable_diffusion/ms_v1_5_pruned_emaonly-d0ab7146.ckpt) converted from torch to MindSpore.
+
 ### 1. Chinese Text-to-Image Generation
```
Comment (SamitHuang) [inline examples/stable_diffusion_v2/README.md:163]: The sub-title should be updated too. 

## Stable Diffusion 1.5

### English Text-to-Image

### Chinese Text-to-Image

...

File: examples/stable_diffusion_v2/README.md [hunk 3]
```diff
@@ -129,18 +129,42 @@ DreamBooth allows users to generate contextualized images of one subject using j
...
 
 ```shell
-# Text to image generation with SD1.x (Support Chinese)
-python text_to_image.py --prompt "雪中之狼"  -v 1.x
+# Text to image generation with SD1.x (Support Chinese & English)
+python text_to_image.py --prompt "雪中之狼"  -v 1.x --ckpt_path PATH_TO_V1_CKPT
```
[the remark this answers is not in the document]
Comment (DongyaoZhu) [reply examples/stable_diffusion_v2/README.md:167]: tokeniser specified in yaml

**Review by congw729 — APPROVED** [review]
lgtm

**Pull request merged**